frontend/src/pages/Support/Support.test.tsx

```diff
@@ -0,0 +1,135 @@
+}));
+
+jest.mock('react-router-dom', () => ({
+	useHistory: useHistoryMock,
+}));
+
+const supportChannels = [
+	{
+		key: 'documentation',
+		name: 'Documentation',
+		icon: <Book />,
+		title: 'Find answers in the documentation.',
+		url: 'https://signoz.io/docs/',
+		btnText: 'Visit docs',
+	},
+	{
+		key: 'github',
+		name: 'Github',
+		icon: <Github />,
+		title: 'Create an issue on GitHub to report bugs or request new features.',
+		url: 'https://github.com/SigNoz/signoz/issues',
+		btnText: 'Create issue',
+	},
+	{
+		key: 'slack_community',
+		name: 'Slack Community',
+		icon: <Slack />,
+		title: 'Get support from the SigNoz community on Slack.',
+		url: 'https://signoz.io/slack',
+		btnText: 'Join Slack',
+	},
+	{
+		key: 'chat',
+		name: 'Chat',
+		icon: <MessageSquare />,
+		title: 'Get quick support directly from the team.',
+		url: '',
+		btnText: launchChat,
+	},
+	{
+		key: 'schedule_call',
+		name: 'Schedule a call',
+		icon: <Calendar />,
+		title: 'Schedule a call with the founders.',
+		url: 'https://calendly.com/pranay-signoz/signoz-intro-calls',
+		btnText: 'Schedule call',
+	},
+	{
+		key: 'slack_connect',
+		name: 'Slack Connect',
+		icon: <Cable />,
+		title: 'Get a dedicated support channel for your team.',
+		url: '',
+		btnText: 'Request Slack connect',
+	},
+];
+
+describe('Help and Support renders correctly', () => {
+	it('should render the support page with all support channels', () => {
+		const { getByText } = render(<Support />);
+		expect(getByText('Support')).toBeInTheDocument();
+		expect(
+			getByText(
+				'We are here to help in case of questions or issues. Pick the channel that is most convenient for you.',
+			),
+		).toBeInTheDocument();
+		supportChannels.forEach((channel) => {
+			expect(getByText(channel.name)).toBeInTheDocument();
+			expect(getByText(channel.title)).toBeInTheDocument();
+			expect(getByText(channel.btnText)).toBeInTheDocument();
+		});
+	});
+	it('should trigger correct handler function when channel button is clicked', () => {
+		const { getByText } = render(<Support />);
+		const button = getByText('Visit docs');
+		const windowOpenMock = jest.spyOn(window, 'open').mockImplementation();
+		fireEvent.click(button);
+		expect(windowOpenMock).toHaveBeenCalledWith(
+			'https://signoz.io/docs/',
+			'_blank',
+		);
+	});
+
+	it('should open Intercom chat widget when Chat channel is clicked', () => {
+		window.Intercom = jest.fn();
+		const { getByText } = render(<Support />);
+		const button = getByText(launchChat);
+		fireEvent.click(button);
+		expect(window.Intercom).toHaveBeenCalledWith('show');
+	});
+});
+
+describe('Handle channels with null or undefined properties', () => {
+	it('should handle window.Intercom being undefined or null', () => {
+		window.Intercom = null;
+		const { getByText } = render(<Support />);
+		const button = getByText(launchChat);
+		fireEvent.click(button);
+		expect(window.Intercom).toBeNull();
```

Comment:
what is the purpose of this test? at line 108 you declared the variable to be null. now even if you skip line 109-111 it will always be null.